Documentation/configuration.md

```diff
@@ -4,6 +4,8 @@ etcd is configurable through command-line flags and environment variables. Optio
 
 The format of environment variable for flag `-my-flag` is `ETCD_MY_FLAG`. It applies to all  flags.
 
+[Official etcd ports](https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?search=etcd) are 2379 for client and 2380 for server. You may find some legacy code still using 4001 for clients and 7001 for servers. Going forward, etcd wants to use 2379 and 2380 as default.
```

Comment:
The [official etcd listening ports]() are 2379 for client requests, and 2380 for peer communication. Some legacy code and documentation still references ports 4001 and 7001, but all new etcd use and discussion should adopt the assigned ports.